### DISCUSSION TITLE
How to hidden feature names and names triggered by the mouse in a track

### DISCUSSION THREAD

**Discussion by DerekZhou817** [GMOD/jbrowse-components#3355] [Q&A]
Hi,

How to hidden feature names and names triggered by the mouse in a track? e.g. In the tract of bed file, chr1:150000, named "mutation".How to hidden the name,"mutation', and "mutation" triggered by the mouse in the track?

Thanks,
Xu

**Comment by garrettjstevens** [comment]
If you mean the mouseover text, you can override it in your track configuration, something like this (see the `"mouseover": ""`):

```json
{
  "type": "FeatureTrack",
  "trackId": "bigbed_genes",
  "name": "BigBed genes",
  "assemblyNames": [
    "volvox"
  ],
  "adapter": {
    "type": "BigBedAdapter",
    "bigBedLocation": {
      "locationType": "UriLocation",
      "uri": "volvox.bb",
      "baseUri": "https://s3.amazonaws.com/jbrowse.org/code/jb2/main/test_data/volvox/config.json"
    }
  },
  "displays": [
    {
      "type": "LinearBasicDisplay",
      "displayId": "bigbed_genes-LinearBasicDisplay",
      "mouseover": ""
    },
  ]
}
```

If that wasn't what you meant, could you post a screenshot showing what you're trying to do?
